todo_app/app.py

```diff
@@ -1,11 +1,19 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', list_items=get_items())
+
+@app.route('/additem', methods=['GET', 'POST'])
+def additem():
+    if request.method == 'POST':
+        add_item(request.form.get('title'))
+        return redirect(url_for('index'))
```

Comment:
Here you say you want to allow GET & POST, but we actually only handle the POST case. I think that latter is good, but we might as well just let Flask handle blocking GETs in that case rather than allowing the app to error as it currently will:
```suggestion
@app.route('/additem', methods=['POST'])
def additem():
    add_item(request.form.get('title'))
    return redirect(url_for('index'))
```